LemonLime.Common/Settings.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace LemonLime.Common
+{
+    public class Settings
+    {
+        public static string SettingsFile;
+
+        // The default settings.
+        private static Dictionary<string, string> DefaultSettings = new Dictionary<string, string>
```

Comment:
Mind making this `private static Dictionary<string, string> DefaultSettings = new Dictionary<string, string> { };`?